misc/twist_mux/launch/twist_mux.launch

```diff
@@ -0,0 +1,33 @@
+<launch>
+  <arg name="robot" default="caffeine"/>
+
```

Comment:
I've mentioned this before but please do not modify 3rd party code (`twist_mux.launch` was not made by us). I believe using it as is and calling it from `keyboard_teleop.launch` with the correct arguments suits our needs.

If you really need to modify their launch file to provide the functionality we need, then duplicate the launch file (rename it) and make the appropriate changes. In this case, we should not modify this launch file, since `keyboard_teleop.launch` never calls it